backend/homework/6/src/main/java/shrishti/example/Main.java

```diff
@@ -0,0 +1,37 @@
+package shrishti.example;
+
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import shrishti.example.beans.speakerService;
+import shrishti.example.beans.tyreService;
+import shrishti.example.config.speakerConfig;
+import shrishti.example.config.tyreConfig;
+import shrishti.example.config.vehicleConfig;
+
+public class Main {
+    public static void main(String[] args) {
+        AnnotationConfigApplicationContext newContext = new AnnotationConfigApplicationContext(new Class[]{vehicleConfig.class, tyreConfig.class, speakerConfig.class});
+
+        speakerService newSpeaker = (speakerService) newContext.getBean("Sony");
+        tyreService newTyre  = (tyreService) newContext.getBean("MRF");
+        vehicleConfig newVehicle = (vehicleConfig) newContext.getBean(vehicleConfig.class);
+        newVehicle.addToVehicleList(newTyre, newSpeaker);
+
+        speakerService newSpeaker2 = (speakerService) newContext.getBean("Bose");
```

Comment:
1. Variables name can be better
2. Make and abstract out a separate randomiser/creator function